capsules/extra/src/ieee802154/device.rs

```diff
@@ -134,11 +134,19 @@ pub trait RxClient {
     /// - `header`: A fully-parsed representation of the MAC header, with the
     /// caveat that the auxiliary security header is still included if the frame
     /// was previously secured.
+    /// - `lqi`: The link quality indicator of the received frame.
```

Comment:
Is LQI actually the correct term here, or should it be RSSI?

https://microchip.my.site.com/s/article/RSSI-vs-LQI-What-is-the-difference

Reply:
Interestingly, both rf233 and nrf52840 are calling this value "LQI" in their spec even though RSSI seems more accurate. 

In the nordic case, the "LQI" is really just the median RSSI of the 3 RSSI values taken during the reception.

Reply:
I guess I'd fall on the side of going with what hardware says then, and stick with LQI.